src/docs/framework-integration/angular.md

```diff
@@ -70,15 +87,43 @@ npm install
 yarn install
 ```
 
-### Adding Angular Output Target
+#### Creating an Angular Component Library
+
+> If you already have an Angular component library, skip this section.
+
+The first time you want to create the component wrappers, you will need to have an Angular library package to write to.
+
+In the `packages/` directory, use the Angular CLI to generate a workspace and a library for your Angular component wrappers:
+
+```bash
+npx -p @angular/cli ng new angular-workspace --no-create-application
+cd angular-workspace
+npx -p @angular/cli ng generate library component-library
+```
+
+You can delete the `component-library.component.ts`, `component-library.service.ts`, and `*.spec.ts` files.
+
+You will also need to add your generated Stencil library as a dependency so import references can be resolved correctly:
+
+```json
+// package.json
+
+"dependencies": {
+  "stencil-library": "*"
+}
+```
```

Comment:
do I need to make this change in `packages/angular-workspace` or in `packages/angular-workspace/projects/component-library` ?